google/cloud/status_or.h

```diff
@@ -176,11 +176,21 @@ class StatusOr final {
    *
    * @param rhs the value used to initialize the object.
    *
-   * @throws only if `T`'s move constructor throws.
+   * @throws ... If `T` move constructor throws.
```

Comment:
Q: why is the apostrophe after backtick bad? Is it only bad in the `@throws` block?

FYI:
```
dbolduc@dbolduc-glaptop:~/code/git/google-cloud-cpp$ grep -rl "\`'" $(git ls-files -- *.h)
docfx/yaml_context.h
google/cloud/bigtable/mutation_batcher.h
google/cloud/mocks/mock_stream_range.h
google/cloud/monitoring/v3/service_monitoring_client.h
google/cloud/options.h
google/cloud/pubsub/blocking_publisher_connection.h
google/cloud/pubsub/internal/session_shutdown_manager.h
google/cloud/pubsub/publisher_connection.h
google/cloud/pubsub/subscriber_connection.h
google/cloud/pubsub/topic_admin_connection.h
google/cloud/pubsublite/internal/alarm_registry_impl.h
google/cloud/spanner/connection.h
google/cloud/spanner/value.h
google/cloud/status_or.h
```

Reply:
Fixed.